src/components/guide/TabDescription.tsx

```diff
@@ -0,0 +1,7 @@
+import { ReactNode } from "react";
```

Comment:
```suggestion
import type { ReactNode } from "react";
```

the new standard for importing types (not functions, classes etc)